docs/DeveloperGuide.md

```diff
@@ -137,13 +135,60 @@ view and manipulate these saved data easily with any available text editor.
 ## 3. Implementation
 This section describes some details on how some features were implemented.
 
+<br/>
+
 ### 3.1 Workspace Feature
 The `Workspace` feature has two core functionalities, to add and switch workspace of the user. 
 
 For adding a new workspace...
 
 
+<br/>
+
 ### 3.2 Estimation Feature
+The `estimate` feature aims to provide translators with better estimates on the time needed to 
+translate a script that they have been assigned. In the past, translators could do this by manually glancing through the script, but they may overlook 
+the amount of words that needs to be translated. 
+Hence, `estimate` helps to ensure that translators **do not overpromise** their clients.
+
+
+#### 3.2.1 Current Implementation
```

Comment:
Is this subsection needed? I wanted to do one for mine as well, but it seems weird without another subsection or "proposed implementation" section

Reply:
I think so because we are gonna have another subsection called design consideration, I guess having just current implementation should be fine!

Reply:
Hmm okay got it! I'll make modifications to my portion in my next PR. Thank you.